src/handler/emulation.rs

```diff
@@ -15,37 +15,42 @@ pub struct EmulationManager {
 }
 
 impl EmulationManager {
-    pub fn init_commands(&mut self, viewport: &Viewport) -> CommandChain {
-        let orientation = if viewport.is_landscape {
-            ScreenOrientation::new(ScreenOrientationType::LandscapePrimary, 90)
-        } else {
-            ScreenOrientation::new(ScreenOrientationType::PortraitPrimary, 0)
-        };
+    pub fn init_commands(&mut self, viewport: &Option<Viewport>) -> Option<CommandChain> {
```

Comment:
Since the `EmulationManager` requires a `Viewport` for its commands, it should definitely require a `&Viewport` instead of `&Option<Viewport>` where a `None` essentially does nothing.
I don't think this change is the right we to address the the optional emulation, instead the `init_commands` should be invoked optionally. So this should be rolled back.

Reply:
That make sense to me.